src/explore-education-statistics-admin/src/pages/release/content/components/ReleaseContent.tsx

```diff
@@ -124,6 +152,12 @@ const ReleaseContent = () => {
 
   return (
     <>
+      <Prompt
+        when={showPrompt}
+        message={() =>
+          'There are unsaved changes.  Clicking away from this tab will result in the changes being lost.'
+        }
+      />
```

Comment:
Think we should be able to do this with a `ModalConfirm`? I haven't delved too deep, but a quick Google seems to surface ways to do this e.g. https://blog.formpl.us/how-to-create-a-custom-route-blocking-modal-in-react-9306139ee3c7

Reply:
Done